libshortfin/docs/conf.py

```diff
@@ -0,0 +1,36 @@
+# Configuration file for the Sphinx documentation builder.
```

Comment:
add license/copyright header?

Reply:
Added a copyright header to the files.

Reply:
Can they use the standard format? https://github.com/nod-ai/SHARK-Platform/blob/19834ee1e1d476a116679afb9521ea0cccdeb308/shortfin/shortfin/framework/session.py#L1-L5

```
# Copyright 2024 Advanced Micro Devices, Inc.
#
# Licensed under the Apache License v2.0 with LLVM Exceptions.
# See https://llvm.org/LICENSE.txt for license information.
# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
```